layouts/partials/navigation.html

```diff
@@ -1,95 +1,131 @@
+              class="fas fa-angle-double-right"></i></a></p>
+      </div>
     </div>
-  </div>
-  {{ if not .IsHome }}
-  <div class="w-100 py-2 breadcrumbBar text-light" style=" {{ if .IsHome }}background-color: transparent{{ else }}background-color: {{ site.Params.text_color_dark }}{{ end }}">  
-    <div class="container">
-      {{ partial "breadcrumb.html" . }}
-      <a data-toggle="collapse" data-target="#sidebarMobile"  aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation"> <i class="ti-menu text-light sidebarIcon"></i></a>
-      <div class="text-light text-center sidebarMobile collapse justify-content-center" id="sidebarMobile">
-        <div class="sidebarBox">
-          <ul>
-            {{ $currentNode := . }}
-            {{range .Site.Home.Sections.ByWeight}}
+    {{ if not .IsHome }}
+    {{ end }}
+  </nav>
+  <nav id="breadcrumbNav">
+    <div class="w-100 py-2 breadcrumbBar text-light"
+      style=" {{ if .IsHome }}background-color: transparent{{ else }}background-color: {{ site.Params.text_color_dark }}{{ end }}">
+      <div class="container">
+        {{ partial "breadcrumb.html" . }}
+        <a data-toggle="collapse" data-target="#sidebarMobile" aria-controls="navigation" aria-expanded="false"
+          aria-label="Toggle navigation"> <i class="ti-menu text-light sidebarIcon"></i></a>
+        <div class="text-light text-center sidebarMobile collapse justify-content-center" id="sidebarMobile">
+          <div class="sidebarBox">
+            <ul>
+              {{ $currentNode := . }}
+              {{range .Site.Home.Sections.ByWeight}}
               {{ if eq .FirstSection $currentNode.FirstSection }}
               {{ template "section-tree-nav" dict "sect" . "currentnode" $currentNode "index" 0}}
               {{ end }}
-            {{ end }}
-            {{ partial "download-pdf.html" .}}
-          </ul>
+              {{ end }}
+              {{ partial "download-pdf.html" .}}
+            </ul>
+          </div>
         </div>
       </div>
+
     </div>
+    <script>
+      var topPosition;
+
+      breadcrumbNav = document.getElementById('breadcrumbNav');
+
+      navbarHeading = document.getElementById('navbarHeading');
+
+      navbarBreadcrumb = document.getElementById('navbarBreadcrumb');
+
+      topPosition = window.pageYOffset;
+
+      window.addEventListener('scroll', function () {
+
+        var scrollPosition = window.pageYOffset || document.documentElement.scrollTop;
+
+        if (scrollPosition > topPosition) {
+          breadcrumbNav.style.display = 'none';
+          navbarHeading.style.display = 'none';
+          navbarBreadcrumb.style.display = 'block';
+        }
 
-  </div>
-  {{ end }}
-</nav>
+        else {
+          breadcrumbNav.style.display = 'block';
+          navbarHeading.style.display = 'block';
+          navbarBreadcrumb.style.display = 'none';
+        }
+      });
+    </script>
```

Comment:
This is simple enough, but I wonder if it would help maintainability if this script were [loaded in `<head>`](https://github.com/unicef/inventory-hugo-theme/blob/cbf2bcf136ef6a9d3b991b0a2e3923cc76837f48/layouts/partials/head.html) instead of here. This way, we could load all JavaScript bits consistently throughout the site, so there is less confusion about how to embed scripts into the DOM.

This change isn't required to merge, but I'd consider it a bonus if you have time to figure out a way to do this similar to other scripts loaded into the DOM.